docs/other.md

```diff
@@ -405,6 +405,14 @@ type 为 all 时，category 参数不支持 cost 和 free
 | ---- | ---- | ----- |
 | yi   | zhu  | shang |
 
+</Route>
```

Comment:
You forgot line 400
`<Route author="LogicJake" example="/gushiwen/recommend/zhushang" path="/gushiwen/recommend/:annotation?"> `

Reply:
Not me, @LogicJake forgot line 400